packages/manager/modules/support/src/tickets/new-ticket/new-ticket.html

```diff
@@ -1,10 +1,12 @@
-<oui-back-button data-state="support.tickets">
+<oui-back-button data-on-click="$ctrl.handleBackButton()">
```

Comment:
```suggestion
<oui-back-button data-on-click="$ctrl.goBack()">
```
? The binding doesn't have to be aware if it's a back button or something else I think